### PR TITLE
Feature/in 409

### PR DESCRIPTION
**Pull request by n3op2**
## IN-409

More -> https://digicatapult.atlassian.net/jira/software/projects/IN/boards/115?selectedIssue=IN-409

#### CLI 'home'
```sh
$ process-management 
Usage: process management [options] [command]

a command line interface for managing chain processes

Options:
  -v, --version                     output current version
  -h, --help                        display help for command

Commands:
  create [options] <json>           A command for persisting process flows onto the chain
  disable [options] <id> <version>  A command for disabling an existing process flows. Required process ID and version
  help [command]                    display help for command
```

#### disable command
```sh
$ process-management help disable
Usage: process management disable [options] <id> <version>

A command for disabling an existing process flows. Required process ID and version

Arguments:
  id                 a valid process id that you would like to disable
  version            a version number of a process

Options:
  --dryRun           to validate process and response locally before persisting on the chain, default - false
  -h, --host <host>  substrate blockchain host address or FQDM, default - "localhost" (default: "localhost")
  -p, --port <port>  specify host port number if it is not a default, default - 9944 (default: "9944")
  -u, --user <user>  specify substrate blockhain user URI, default - "//Alice" (default: "//Alice")
  --help             display help for command
$ 
```

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -26,50 +30,64 @@ program: [
...
+      const { dryRun } = options
+      const res: Process.Result = await disableProcess(id, parseInt(version), dryRun, mapOptions(options))
+      log(` ${g('command [disable] executed successfully')}: ${JSON.stringify(res)}`)
+
+      process.exit(1)
+    } catch (err) {
+      log(`Exception has been caught: ${r(JSON.stringify(err))}`)
+      program.help()
+    }
+    log('not implemented', { id, version, options })
```
Comment (jonmattgray) [inline src/index.ts:90]: can remove this log

Comment (n3op2) [reply src/index.ts:90]: oops:), thanks

File: README.md [hunk 2]
```diff
@@ -68,7 +68,8 @@ Commands:
   help [command]             display help for command
 ```
 
-### Create Process Command
+
+### Crete Process Command
```
Comment (jonmattgray) [inline README.md:72]: typo

Comment (n3op2) [reply README.md]: must have accidentally deleted, as this line should have not been changed, thanks for flagging, as always!

**Review by mattdean-digicatapult — COMMENTED** [review]
I think we're we're missing some integration tests for disable?

**Review by mattdean-digicatapult — APPROVED** [review]
lgtm

**Pull request merged**